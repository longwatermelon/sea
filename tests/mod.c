fn main() -> int {
    return 5%2;
}
