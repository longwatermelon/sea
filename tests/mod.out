RETURN_CODE:1
STDOUT:
